### PR TITLE
options to disable/limit pooling + fix influxdb write

### PR DESCRIPTION
**Pull request by fingon**
What we're actually already using (see commits for content)

### CODE REVIEW

File: src/x/pool/options.go [hunk 1]
```diff
@@ -36,10 +42,21 @@ type objectPoolOptions struct {
 	onPoolAccessErrorFn OnPoolAccessErrorFn
 }
 
+func getDefaultSize() int {
+	v, ok := os.LookupEnv("AIVEN_POOL_DEFAULT")
+	if ok {
```
Comment (tsharju) [inline src/x/pool/options.go:47]: I've grown to like the "comma ok" idiom _(a.k.a someones been reading "effective go")_... So maybe...
```golang
if v, ok := os.LookupEnv("AIVEN_POOL_DEFAULT"); ok {
...
}`

File: src/x/pool/options.go [hunk 2]
```diff
@@ -49,7 +66,19 @@ func NewObjectPoolOptions() ObjectPoolOptions {
 
 func (o *objectPoolOptions) SetSize(value int) ObjectPoolOptions {
 	opts := *o
-	opts.size = value
+	vlist, ok := os.LookupEnv("AIVEN_POOL_VALID")
+	if ok {
```
Comment (tsharju) [inline src/x/pool/options.go:70]: The same "comma ok" thing as above.

**Review by tsharju — CHANGES_REQUESTED** [review]
Some nitpicking... otherwise looks good to me.

File: src/x/pool/options.go (hunk 1 above)
Comment (fingon) [reply src/x/pool/options.go:47]: Personally not super fond of the format as it is easy to have errors on too long lines, but don't care much either way either so changing 😏

**Review by tsharju — APPROVED** [review]

**Pull request merged**